pkg/build/packaging/grafana.go

```diff
@@ -379,6 +379,9 @@ func executeFPM(options linuxPackageOptions, packageRoot, srcDir string) error {
 		"--vendor", vendor,
 		"-a", string(options.packageArch),
 	}
+	if options.prermSrc != nil && options.prermSrc != "" {
```

Comment:
```suggestion
	if options.prermSrc != "" {
```
this can never be nil so we can just check it's not empty